configure.ac

```diff
@@ -804,24 +806,33 @@ else
 	*-*-solaris*)
 		case $PTHREAD_CC in
 		*gcc*)
-			VCC_CC="$PTHREAD_CC $OCFLAGS $PTHREAD_CFLAGS -fpic -shared -o %o %s"
```

Comment:
I would not be surprised to see fallout from some of the shell variables containing `%`, but I guess we can handle it when it happens?

Reply:
I think so too, but I also did 60143b401d1be5020f5bd4d6e607407b7d8ed2b3 when I noticed, even though those percent sequences were no-ops.